lms/static/js/views/fields.js

```diff
@@ -123,7 +136,8 @@
                 this.lastSuccessMessageContext = context;
 
                 setTimeout(function () {
-                    if ((context === view.lastSuccessMessageContext) && (view.getNotificationMessage() === successMessage)) {
+                    if ((context === view.lastSuccessMessageContext) &&
+                        (view.getNotificationMessage() === successMessage.toString())) {
```

Comment:
I guess this is one of those dangerous spots.  This should probably be mentioned in the docs.